lib/plugins/aws/info/getResourceCount.test.js

```diff
@@ -0,0 +1,118 @@
+        LastUpdatedTimestamp: '2018-03-12T00:22:40.680Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'ApiGatewayMethodHelloGet',
+        PhysicalResourceId: 'hello-ApiGa-11R27BUE48W38',
+        ResourceType: 'AWS::ApiGateway::Method',
+        LastUpdatedTimestamp: '2018-03-12T00:22:37.478Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'ApiGatewayResourceHello',
+        PhysicalResourceId: 'az5f7l',
+        ResourceType: 'AWS::ApiGateway::Resource',
+        LastUpdatedTimestamp: '2018-03-12T00:22:22.916Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'ApiGatewayRestApi',
+        PhysicalResourceId: 'n1uk4p7kl0',
+        ResourceType: 'AWS::ApiGateway::RestApi',
+        LastUpdatedTimestamp: '2018-03-12T00:22:19.768Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'HelloLambdaFunction',
+        PhysicalResourceId: 'hello-world-2-dev-hello',
+        ResourceType: 'AWS::Lambda::Function',
+        LastUpdatedTimestamp: '2018-03-12T00:22:34.095Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'HelloLambdaPermissionApiGateway',
+        PhysicalResourceId: 'hello-world-2-dev-HelloLambdaPermissionApiGateway-18KKZXJG1DPF5',
+        ResourceType: 'AWS::Lambda::Permission',
+        LastUpdatedTimestamp: '2018-03-12T00:22:46.950Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'HelloLambdaVersiongZDaMtQjEhvXacHdpTLnQ61zDCdI2IWVYCbuE50pj8',
+        PhysicalResourceId: 'arn:aws:lambda:us-east-1:111111111:function:hello-world-2-dev-hello:2',
+        ResourceType: 'AWS::Lambda::Version',
+        LastUpdatedTimestamp: '2018-03-12T00:22:37.256Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'HelloLogGroup',
+        PhysicalResourceId: '/aws/lambda/hello-world-2-dev-hello',
+        ResourceType: 'AWS::Logs::LogGroup',
+        LastUpdatedTimestamp: '2018-03-12T00:22:20.095Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'IamRoleLambdaExecution',
+        PhysicalResourceId: 'hello-world-2-dev-us-east-1-lambdaRole',
+        ResourceType: 'AWS::IAM::Role',
+        LastUpdatedTimestamp: '2018-03-12T00:22:30.995Z',
+        ResourceStatus: 'CREATE_COMPLETE' },
+      { LogicalResourceId: 'ServerlessDeploymentBucket',
+        PhysicalResourceId: 'hello-world-2-dev-serverlessdeploymentbucket-1e3l68m8zaz7i',
+        ResourceType: 'AWS::S3::Bucket',
+        LastUpdatedTimestamp: '2018-03-12T00:22:11.380Z',
+        ResourceStatus: 'CREATE_COMPLETE' }],
+    };
+
+    listStackResourcesStub.resolves(listStackResourcesResponse);
+
+    awsInfo.gatheredData = {
+      info: {
+        functions: [],
+        endpoint: '',
+        service: '',
+        stage: '',
+        region: '',
+        stack: '',
+      },
+      outputs: [],
+    };
+
+    return awsInfo.getResourceCount().then(() => {
```

Comment:
Use `chai-as-promised` style like this https://github.com/serverless/serverless/blob/master/lib/plugins/plugin/plugin.test.js

Reply:
@horike37 I don't quite understand this one. Which line is a good example in the `plugin.test.js`?

Reply:
@alexdebrie 
Please see this line. You can test if it returns `fulfilled` or `rejected` as a promise by using `chai-as-promised`. Otherwise, you can't handle the test process if the current test is and returns `rejected`.
https://github.com/serverless/serverless/blob/master/lib/plugins/plugin/plugin.test.js#L55

Reply:
@horike37 Thanks! I gave it a shot -- let me know if that's right 😄